tests/nightly/test_large_array.py

```diff
@@ -1199,6 +1185,118 @@ def test_full():
     assert a[-1][-1] == 3
 
 
+def test_hyperbolic():
+    def test_arccosh(a):
+        mx_res = mx.nd.arccosh(a)
+        assert_almost_equal(mx_res[-1][-1].asnumpy(), np.arccosh(a[-1][-1].asnumpy()))
+
+    def test_arcsinh(a):
+        mx_res = mx.nd.arcsinh(a)
+        assert_almost_equal(mx_res[-1][-1].asnumpy(), np.arcsinh(a[-1][-1].asnumpy()))
+
+    def test_arctanh(a):
+        a[-1][-1] = 0   # arctanh of 1 is inf, assert_almost_equal gives "divide by 0" for comparing 2 inf values
+        mx_res = mx.nd.arctanh(a)
+        assert_almost_equal(mx_res[-1][-1].asnumpy(), np.arctanh(a[-1][-1].asnumpy()))
+
```

Comment:
can this be merged inside the trignometric functions added by : https://github.com/apache/incubator-mxnet/pull/16477

Reply:
initial work division was done in this way. Can we do it later in a separate PR ?

Reply:
please do it in one of the four PRs that you two have opened. Shouldnt be very difficult. Let me know if this is the case.

Reply:
merged